src/DropboxAdapter.php

```diff
@@ -182,11 +182,39 @@ public function listContents($directory = '', $recursive = false): array
             return [];
         }
 
-        return array_map(function ($entry) {
+        $cleanedPathDisplay = [];
```

Comment:
Generally speaking I don't look for loops and temporary variables. Could you refactor this code to use `array_map` and related functions?